numba/array.py

```diff
@@ -0,0 +1,177 @@
+from numba.tracing import trace
+import numpy
+import sys
+
+def _o2s(dtype, shape, order):
+    # convert order parameter to strides
+
+    if dtype is None or shape is None or order is None:
```

Comment:
Actually, I retract my earlier statement. I tried to find code in NumPy that shows how 'None' arguments are handled (i.e., what the defaults are), but didn't find anything. Given that both `_s2o` and `_o2s` are private functions, I think it's safe to leave them as the are.